### PR TITLE
Add AstraDB tile

### PR DESCRIPTION
**Pull request by annthurium**
Ready for review!

### CODE REVIEW

**Review by TuanaCelik — APPROVED** [review]
Looks great @annthurium - I've made a change here, making the logo white instead because it'll look better on the tile. If you're ok with that, merge away

File: integrations/astradb.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+						Document(content="In certain parts of the world, like the Maldives, Puerto Rico, and San Diego, you can witness the phenomenon of bioluminescent waves.")]
+
+document_embedder = SentenceTransformersDocumentEmbedder(model_name_or_path=model_name_or_path)  
+document_embedder.warm_up()
+documents_with_embeddings = document_embedder.run(documents)
+
+document_store.write_documents(documents_with_embeddings.get("documents"), policy=DuplicatePolicy.SKIP)
+
+query_pipeline = Pipeline()
+query_pipeline.add_component("text_embedder", SentenceTransformersTextEmbedder(model_name_or_path=model_name_or_path))
```
Comment (TuanaCelik) [inline integrations/astradb.md:127]:
```suggestion
query_pipeline.add_component("text_embedder", SentenceTransformersTextEmbedder(model=model_name_or_path))
```

File: integrations/astradb.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    astra_application_token=astra_application_token,
+)
+
+model_name_or_path = "sentence-transformers/all-mpnet-base-v2"
+
+documents = [Document(content="There are over 7,000 languages spoken around the world today."),
+						Document(content="Elephants have been observed to behave in a way that indicates a high level of self-awareness, such as recognizing themselves in mirrors."),
+						Document(content="In certain parts of the world, like the Maldives, Puerto Rico, and San Diego, you can witness the phenomenon of bioluminescent waves.")]
+
+document_embedder = SentenceTransformersDocumentEmbedder(model_name_or_path=model_name_or_path)
```
Comment (TuanaCelik) [inline integrations/astradb.md:120]:
```suggestion
document_embedder = SentenceTransformersDocumentEmbedder(model=model_name_or_path)  
```

File: integrations/astradb.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+document_store = AstraDocumentStore(
+    astra_id=astra_id,
+    astra_region=astra_region,
+    astra_collection=collection_name,
+    astra_keyspace=keyspace_name,
+    astra_application_token=astra_application_token,
+)
+
+model_name_or_path = "sentence-transformers/all-mpnet-base-v2"
```
Comment (TuanaCelik) [inline integrations/astradb.md:114]:
```suggestion
model = "sentence-transformers/all-mpnet-base-v2"
```

File: integrations/astradb.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+
+documents = [Document(content="There are over 7,000 languages spoken around the world today."),
+						Document(content="Elephants have been observed to behave in a way that indicates a high level of self-awareness, such as recognizing themselves in mirrors."),
+						Document(content="In certain parts of the world, like the Maldives, Puerto Rico, and San Diego, you can witness the phenomenon of bioluminescent waves.")]
+
+document_embedder = SentenceTransformersDocumentEmbedder(model_name_or_path=model_name_or_path)  
+document_embedder.warm_up()
+documents_with_embeddings = document_embedder.run(documents)
+
+document_store.write_documents(documents_with_embeddings.get("documents"), policy=DuplicatePolicy.SKIP)
```
Comment (TuanaCelik) [inline integrations/astradb.md:124]: Can we just check that we don't have to import DuplicatePolicy to have this line work?

Comment (annthurium) [reply integrations/astradb.md:124]: oh yeah it's not necessary, it's just a convenience for testing if you want to run the code multiple times. I'll remove it. thanks!

**Pull request merged**